operator/src/main/java/oracle/kubernetes/operator/Main.java

```diff
@@ -333,38 +327,41 @@ static Step createDomainRecheckSteps(DateTime now) {
     return strategy;
   }
 
-  static Step readExistingResources(String operatorNamespace, String ns) {
-    return Step.chain(
-        new ReadExistingResourcesBeforeStep(),
-        ConfigMapHelper.createScriptConfigMapStep(operatorNamespace, ns),
-        createConfigMapStep(ns),
-        readExistingPods(ns),
-        readExistingEvents(ns),
-        readExistingServices(ns),
-        readExistingDomains(ns));
-  }
+  static Step readExistingResources(String ns) {
+    NamespacedResources resources = new NamespacedResources(ns, null);
+    DomainResourcesValidation dpis = new DomainResourcesValidation(ns, processor);
```

Comment:
Should the variable name 'dpis' be something like 'drv' so it's not confused with DomainPresenceInfo?

Reply:
That's a good idea.